### PR TITLE
ARUHA-310: removed extra logging of ATs

### PR DESCRIPTION
**Pull request by v-stepanov**
@antban @v1ctor @rcillo @adyach please review

### CODE REVIEW

**Comment by v1ctor** [comment]
:shipit:

File: build.gradle [hunk 1]
```diff
@@ -285,8 +285,6 @@ acceptanceTest {
     testLogging {
         events "passed", "skipped", "failed"
     }
-    testLogging.showStandardStreams = true
-    testLogging.exceptionFormat = 'full'
```
Comment (adyach) [inline build.gradle]: @v-stepanov maybe we can at least leave 'testLogging.exceptionFormat = 'full'? I would like to see the reason of the failing test when I run tests from console.

Comment (v-stepanov) [inline build.gradle]: You're right. I returned it back.

**Comment by adyach** [comment]
👍

**Comment by adyach** [comment]
👍

**Pull request merged**